tests/test_redshift.py

```diff
@@ -182,6 +182,20 @@ def test_redshift_snapshot(self):
         resources = p.run()
         self.assertEqual(len(resources), 1)
 
+    def test_redshift_enhanced_vpc_routing(self):
+        factory = self.replay_flight_data('test_redshift_enhanced_vpc_routing')
+        p = self.load_policy({
+            'name': 'redshift-enhanced-vpc-routing',
+            'resource': 'redshift',
+            'filters': [
+                {'type': 'value',
+                 'key': 'EnhancedVpcRouting',
+                 'value': True}],
+            'actions': [{'type': 'enhanced-vpc-routing', 'enabled': False}]},
+            session_factory=factory)
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
```

Comment:
querying and asserting that modified status would suffice. else i could completely change the implementation to do nothing and the test still passes. you don't have to fake it, you can query post the policy execution for the actual. the same placebo session factory thats being passed to the load policy can also be used to construct a client for the tests to create or verify scenarios.